src/applications/caregivers/definitions/UIDefinitions/sharedUI.js

```diff
@@ -35,9 +39,10 @@ export const genderUI = label => ({
 
 export const fullNameUI = label => ({
   first: {
-    'ui:title': `${label} first name`,
+    'ui:title': `${label} legal first name`,
+    'ui:description': VeteranFullNameDescription,
```

Comment:
You're going to want to take this out of here and extend the definition in the ui:schema declaration. This is used whenever there is a name field (like for primary and secondary caregiver) so the veteran description will not apply in those cases. See how the SSN is extended in `vetInfo.js` form config file.

Reply:
Hey Matt let me know how this looks. I made it more dynamic based on Jerek's suggestion. Now the new customFullNameUI can extend a specific field within a UI schema object with additional properties.